pigeon/notifications.dart

```diff
@@ -26,3 +26,16 @@ abstract class NotificationHostApi {
   /// See: https://developer.apple.com/documentation/uikit/uiapplication/launchoptionskey/remotenotification
   NotificationPayloadForOpen? getNotificationDataFromLaunch();
 }
+
+@EventChannelApi()
+abstract class NotificationHostEvents {
```

Comment:
Would `NotificationEventChannelApi` be a better name for this, to make it clearer what kind of thing it is?